test/core/end2end/tests/shutdown_finishes_calls.c

```diff
@@ -159,7 +159,7 @@ static void test_early_server_shutdown_finishes_inflight_calls(
 
   grpc_server_destroy(f.server);
 
-  GPR_ASSERT(status == GRPC_STATUS_UNAVAILABLE);
+  GPR_ASSERT(status == GRPC_STATUS_INTERNAL || status == GRPC_STATUS_UNAVAILABLE);
```

Comment:
Why has this one become ambiguous. Is it timing dependent? Add comment if so

Reply:
Added a comment